src/utils/web.py

```diff
@@ -73,6 +77,18 @@ def urlencode(*args, **kwargs):
     from urllib.error import HTTPError, URLError
     from urllib.parse import splithost, splituser
 
+if svi[0] == 2 and svi[1] == 7 and svi[2] >= 9:
+    import ssl
+    context = ssl._create_unverified_context()
+    ca_file = os.path.exists(ssl.get_default_verify_paths().openssl_cafile)
+elif svi[0] >= 3:
+    import ssl
+    context = ssl._create_unverified_context()
+    ca_file = os.path.exists(ssl.get_default_verify_paths().openssl_cafile)
```

Comment:
If you apply the above change, you should be able to remove this `elif` block entirely because `(3, X, Y) >= (2, 7, 9)`